scripts/push-image.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ "$#" != 1 ]
+then
+  echo "Usage: bash $0 BRANCH"
+  exit 1
+fi
+
+BRANCH=$1
```

Comment:
This is done in the Makefile.  Please remove this from the build script

Reply:
Sure. I will close this PR, pull the latest from master branch and create a new one. Thanks.